src/IBM.Cloud.SDK.Core/Service/IBMService.cs

```diff
@@ -19,37 +19,25 @@
 using System.Collections.Generic;
 using System.Net.Http;
 using IBM.Cloud.SDK.Core.Authentication;
-using IBM.Cloud.SDK.Core.Authentication.BasicAuth;
-using IBM.Cloud.SDK.Core.Authentication.Iam;
-using IBM.Cloud.SDK.Core.Authentication.Icp4d;
 using IBM.Cloud.SDK.Core.Authentication.Noauth;
 using IBM.Cloud.SDK.Core.Http;
 using IBM.Cloud.SDK.Core.Util;
-
+
 namespace IBM.Cloud.SDK.Core.Service
 {
     public abstract class IBMService : IIBMService
     {
-        //const string PATH_AUTHORIZATION_V1_TOKEN = "/authorization/api/v1/token";
+        public static string propnameUrl = "URL";
+        public static string propnameDisableSsl = "DISABLE_SSL";
+
         private const string icpPrefix = "icp-";
         private const string apikeyAsUsername = "apikey";
-        public string SERVICE_NAME;
-        public string ApiKey { get; set; }
+        public string serviceName;
         public IClient Client { get; set; }
-        private bool skipAuthentication = false;
-        public bool SkipAuthentication
-        {
-            get { return skipAuthentication; }
-            set
-            {
-                skipAuthentication = value;
-                authenticator = new NoauthAuthenticator(null);
-            }
-        }
-        public string DefaultEndpoint { get; }
         public string ServiceName { get; set; }
         public string Url { get { return Endpoint; } }
         protected Dictionary<string, string> customRequestHeaders = new Dictionary<string, string>();
+        private const string errormsgNoAuthenticator = "Authentication information was not properly configured.";
```

Comment:
how about `NoAuthErrorMessage` for variable name?

Reply:
never mind I just noticed you are using the above format across SDK so better to stay consistent.